build.gradle

```diff
@@ -111,6 +111,7 @@ ext {
       project(':h2o-jetty-9'),
       project(':h2o-ext-jython-cfunc'),
       project(':h2o-hive'),
+      project(':h2o-persist-s3-common'),
```

Comment:
should be in published projects as well, it can also be just in h2o-core - it will never be excluded from any assembly

Reply:
I made it part of core. Finally makes sense to me. You were right from the beginning.